source/Octopus.Cli/Commands/Releases/CreateReleaseCommand.cs

```diff
@@ -43,6 +43,7 @@ public class CreateReleaseCommand : DeploymentCommandBase, ISupportFormattedOutp
             var options = Options.For("Release creation");
             options.Add<string>("project=", "Name or ID of the project.", v => ProjectNameOrId = v);
             options.Add<string>("defaultPackageVersion=|packageVersion=", "Default version number of all packages to use for this release. Override per-package using --package.", versionResolver.Default);
+            options.Add<string>("gitCommit=", "[Optional, Experimental] Git commit to use when creating the release. Use in conjunction with the --gitRef parameter.", v => GitCommit = v);
```

Comment:
```suggestion
            options.Add<string>("gitCommit=", "[Optional, Experimental] Git commit to use when creating the release. Use in conjunction with the --gitRef parameter the select any previous commit (not limited to `HEAD~0).", v => GitCommit = v);
```

I think a little elaboration here could help, what do you think?

Reply:
Added a bit extra sans calling out `HEAD~0` since we haven't explicitly catered for that format  _yet_ :)